pages/index.js

```diff
@@ -1,25 +1,64 @@
 import GridOfCards from "@/components/GridOfCards";
 import Navigation from "@/components/Navigation";
+import { WinIcon } from "@/components/SVG/win";
 import { StyledButton } from "@/components/StyledButton";
+import { styled } from "styled-components";
 
 export default function HomePage({
   handleActiveCards,
   shuffledCards,
   shuffle,
 }) {
+  const firstHorizontalRowWin =
+    shuffledCards[0]?.isActive &&
+    shuffledCards[1]?.isActive &&
+    shuffledCards[2]?.isActive;
+  const secondHorizontalRowWin =
+    shuffledCards[3]?.isActive &&
+    shuffledCards[4]?.isActive &&
+    shuffledCards[5]?.isActive;
+  const thirdHorizontalRowWin =
+    shuffledCards[6]?.isActive &&
+    shuffledCards[7]?.isActive &&
+    shuffledCards[8]?.isActive;
+  const firstHorizontalColumWin =
+    shuffledCards[0]?.isActive &&
+    shuffledCards[3]?.isActive &&
+    shuffledCards[6]?.isActive;
+  const secondHorizontalColumWin =
+    shuffledCards[1]?.isActive &&
+    shuffledCards[4]?.isActive &&
+    shuffledCards[7]?.isActive;
+  const thirdHorizontalColumWin =
+    shuffledCards[2]?.isActive &&
+    shuffledCards[5]?.isActive &&
+    shuffledCards[8]?.isActive;
+  const bingo =
+    firstHorizontalRowWin ||
+    secondHorizontalRowWin ||
+    thirdHorizontalRowWin ||
+    firstHorizontalColumWin ||
+    secondHorizontalColumWin ||
+    thirdHorizontalColumWin;
```

Comment:
okay thats one way :D maybe in the future you find a shorter solution. For now its fine 👍

Reply:
what about the diagonals?